package.json

```diff
@@ -7,7 +7,7 @@
     "all": "run-s lint test",
     "lint": "eslint .",
     "dev": "npm test -- --watch",
-    "test": "mocha 'test/**/*.js'",
+    "test": "mocha test/rules/**/*.js test/**/*-spec.js",
```

Comment:
What exactly is this for?

Reply:
I wanted to exclude the utils from being run. But maybe it's a better idea to always use `-spec` suffix.

Reply:
Reorganized via https://github.com/camunda/bpmnlint-plugin-camunda/pull/11/commits/5c5da9871b1f30917f4858f4add1fca01b3d6e2d